CHANGELOG.md

```diff
@@ -7,12 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
```

Comment:
```suggestion
## [0.5.1]
```

Reply:
We either need to change the manual instructions to make this update manually, or adopt the `update-changelog` script we have in some of the other repos.

Reply:
Manual instructions are up-to-date:
```markdown
4. Update the CHANGELOG.md file, adding your changes in a section **beneath the new release number**.
```

Reply:
Maybe we need a step 'add a new release number if one is needed' for clarity